activesupport/lib/active_support/inflector/inflections.rb

```diff
@@ -1,4 +1,6 @@
-require 'concurrent'
+# FIXME: clean up warning spam from concurrent lib
```

Comment:
The warnings are fixed, except for https://github.com/ruby-concurrency/concurrent-ruby/pull/419